spec/serializer_spec.rb

```diff
@@ -768,4 +768,36 @@ def serialize_primary(object, options = {})
       })
     end
   end
+
+  describe 'inheritance through subclassing' do
+    it 'inherits attributes' do
+      tagged_post = create(:tagged_post)
+      options = {serializer: MyApp::PostSerializerWithInheritedProperties}
+      data = JSONAPI::Serializer.serialize(tagged_post, options);
+      expect(data['data']['attributes']).to have_key('title');
```

Comment:
I'd rather this assert `eq(..)` to the actual content of the title, the factories create deterministic data for this reason.

Also, I you add the "tag" attribute to the subclass but I don't believe you test for it anywhere.